i18n-helpers/Cargo.toml

```diff
@@ -19,6 +19,7 @@ pulldown-cmark-to-cmark = "11.0.0"
 regex = "1.9.4"
 semver = "1.0.16"
 serde_json = "1.0.91"
+test-case = "3.1.0"
```

Comment:
This is a test dependency, right? Then it should go into the `dev-dependencies` below.

However, I don't think we need extra complex machinery like this for our tests. Keeping things simple is normally better (the discussion about `&'static str` makes me feel that the new code isn't simple).